### PR TITLE
CLDSRV-34 - fix time computation for test comparison

### PR DESCRIPTION
**Pull request by lavigne958**
need a new date that is 30 minutes in the past.
Computation is based on a date already 30 minutes in the future,
take that date and rewind it by 60 minutes
- 30 to go back to the original data
- 30 to go back 30 minutes in the past

# Pull request template

### Motivation and context

This change is required as it brakes a unit test

### CODE REVIEW

**Comment by lavigne958** [comment]
Hey bert-e

File: tests/unit/api/objectHead.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const incorrectMD5 = 'fkjwelfjlslfksdfsdfsdfsdfsdfsdj';
 const objectName = 'objectName';
 const date = new Date();
 const laterDate = date.setMinutes(date.getMinutes() + 30);
-const earlierDate = date.setMinutes(date.getMinutes() - 30);
+const earlierDate = date.setMinutes(date.getMinutes() - 60);
```
Comment (miniscruff) [inline tests/unit/api/objectHead.js]: I think the problem here is that date is being changed with each method as setMinutes updates in place.
You probably want to do `laterDate = new Date().setMinutes(...);` and so on.

**Review by miniscruff — CHANGES_REQUESTED** [review]

File: tests/unit/api/objectHead.js (hunk 1 above)
Comment (miniscruff) [reply tests/unit/api/objectHead.js]: Actually on second thought I think just doing two calls is better.
```js
const earlierDate = new Date();
earlierDate.setMinutes(...);
```

Comment (lavigne958) [reply tests/unit/api/objectHead.js]: Right it actually looks easier to read and understand.

**Review by miniscruff — APPROVED** [review]

**Review by XinLi0207 — APPROVED** [review]

**Comment by lavigne958** [comment]
squashed commits and rebased PR.

**Pull request merged**